docs/write-a-plugin.md

```diff
@@ -18,9 +18,21 @@ window.$docsify = {
         return content;
       });
 
+      hook.beforeEmbed(function(tokens, next) {
+        // Invoked each time after parsing the Markdown file, but before embedding is applied to the token list.
```

Comment:
I think we need to change the wording, or otherwise describe what "embedding" is. I don't think that currently Docsify users have any concept of "embed", and the term "embed" is currently just an internal detail.

Reply:
I doubt that given there is a whole page about embedding files in the documentation: https://docsify.js.org/#/embed-files

Reply:
Oh, yeah, oops. 😄

Does marked give back tokens just for the embed? Or what exactly?

Reply:
"tokens" is an array of token, produced by the `compile.lexer(raw)` call. It is indeed a vendor-specific data structure produced by marked.

https://github.com/docsifyjs/docsify/blob/66303fec4c7115621e556ad742cfac9d19f26bd9/src/core/render/embed.js#L103

Docsify's embed mechanism operates on this token list as well.